resources/report-table-formatter.js

```diff
@@ -1,50 +1,50 @@
-{
-    "Lineage": function format(value) {
-        $(function () {
-            $('[data-toggle="tooltip"]').tooltip()
-        })
+var formatter = {
+  Lineage: function format(value) {
+    $(function () {
+      $('[data-toggle="tooltip"]').tooltip();
+    });
```

Comment:
Especially these changes. Not sure if they still produce the same output. We should have an eye on that